packages/common/src/httpRequests.ts

```diff
@@ -39,7 +39,7 @@ export async function sendRequest<T>({ url, method, body, headers = {} }: HttpRe
       return jsonResponse as T
     } else if (jsonResponse && jsonResponse.hasOwnProperty('error')) {
       const error = jsonResponse.error
-      throw new Error(`JSON-RPC error: code ${error.code}, message: ${error.message}`)
+      throw new Error(`JSON-RPC error: ${error}`)
```

Comment:
This way of throwing is not useful to the developers when they do error handling on their side. Either throw a proper object in Error class instead of just a single string, so developers can do proper error handling or throw the original error object returned in json rpc response.

Reply:
When you'll write test case for this method, then you'll see the use of this error handling.

Reply:
now sending the error object